### PR TITLE
fix: cannot show the dummy cells at the farthest right in scrollX

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* related issue(#1288)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jung-han — APPROVED** [review]
완료입니다~

File: packages/toast-ui.grid/cypress/integration/dummyRows.spec.ts [hunk 1]
```diff
@@ -1,35 +1,68 @@
...
+  });
 
-it('should render dummy rows', () => {
-  cy.getByCls('cell-dummy').should('exist');
-});
+  it('should render dummy rows', () => {
+    cy.getByCls('cell-dummy').should('exist');
+  });
+
+  it('should inialize focus and selection layer', () => {
```
Comment (shiren) [inline packages/toast-ui.grid/cypress/integration/dummyRows.spec.ts:21]: initialize

**Review by shiren — APPROVED** [review]
수고하셨습니다.

**Review by lja1018 — APPROVED** [review]
리뷰완료입니다.

**Review by jungeun-cho — APPROVED** [review]
확인했습니다.

**Review by dotaitch — APPROVED** [review]
리뷰 완료합니다.

**Pull request merged**